.github/workflows/python-app.yml

```diff
@@ -23,7 +23,19 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8
+        pip install flake8 pytest
+        pip install -r requirements-dev.txt
     - name: Lint with flake8
       run: |
-        flake8 . --count --show-source --statistics
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
there is a .flake8 file in the repo so are these settings needed?

Reply:
you're right - if it ain't broke

reverted that